query/src/main/java/io/keen/client/java/RelativeTimeframe.java

```diff
@@ -0,0 +1,47 @@
+package io.keen.client.java;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Relative Timeframe, as specified by API docs: https://keen.io/docs/api/#relative-timeframes
+ *
+ * Created by claireyoung on 7/6/15.
+ */
+class RelativeTimeframe implements Timeframe {
+
+    private final String relativeTimeframe;
+
+    /**
+     * Construct an AbsoluteTimeframe with a specified start and end date/time.
+     * @param relativeTimeframe the relative timeframe string, as specified by the API docs.
+     *                          For example, "this_minute", "this_month", "this_year", etc.
+     */
+    public RelativeTimeframe(String relativeTimeframe) {
+        this.relativeTimeframe = relativeTimeframe;
+    }
+
+    /**
+     * Get the relative timeframe.
+     *
+     * @return  the relative timeframe.
+     */
+    public String getTimeframe() {
+        return relativeTimeframe;
+    }
+
+    /**
+     * Construct the Timeframe map to send in the query.
+     *
+     * @return  the Timeframe Json map to send in the query.
+     */
+    public Map<String, Object> constructTimeframeArgs() {
```

Comment:
Nit: Since this is implementing an interface method, it should be marked with `@Override`.